localstack-core/localstack/services/apigateway/next_gen/execute_api/variables.py

```diff
@@ -179,3 +185,20 @@ class LoggingContextVariables(TypedDict, total=False):
     status: Optional[str]
     waf: Optional[LoggingContextVarsWaf]
     xrayTraceId: Optional[str]
+
+
+class MappingTemplateParams(TypedDict, total=False):
+    path: dict[str, str]
+    querystring: dict[str, str]
+    header: dict[str, str]
+
+
+class MappingTemplateInput(TypedDict):
+    body: str
+    params: MappingTemplateParams
+
+
+class MappingTemplateVariables(TypedDict, total=False):
+    context: ContextVariables
+    input: MappingTemplateInput
+    stageVariables: dict[str, str]
```

Comment:
Not too sure this should be here, maybe the file itself is too restrictively named, but I would maybe put this in the `template_mapping.py`, as it's only related to it?

Reply:
Agreed, I wasn't too happy to put it there either. `template_mapping.py` seems better! :+1: